src/frontends/tensorflow_common/src/op/atan2.cpp

```diff
@@ -0,0 +1,71 @@
+#include "common_op_table.hpp"
+#include "common_op_table.hpp"
+#include "openvino/op/atan.hpp"
```

Comment:
Please format your code based on clang rules. Your IDE should apply these rules.